connector/connector_ldap.go

```diff
@@ -154,11 +157,29 @@ func (cfg *LDAPConnectorConfig) Connector(ns url.URL, lf oidc.LoginFunc, tpls *t
 		tlsConfig.Certificates = []tls.Certificate{cert}
 	}
 
+	var initConn, maxConn int
+	if cfg.InitConn != 0 {
+		initConn = cfg.InitConn
+	} else {
+		initConn = 5
+	}
+	if cfg.MaxConn != 0 {
```

Comment:
Yes, there will be constants. I'll post a follow-up PR to turn the other defaults into constants as well.